doc/userguide/rules/intro.rst

```diff
@@ -252,6 +257,25 @@ as the direction specifies that we do not want to evaluate the response packet.
 
    There is no 'reverse' style direction, i.e. there is no ``<-``.
 
+Here is an example of a bidirectional rule:
```

Comment:
This is very confusing to me. `bidirectional` and `both dir` seem to be interchangeably used throughout the code.
The pre-existing `<>` operator uses flags like `SIG_FLAG_INIT_BIDIREC`.. I'm unable to decipher the difference. Could you please explain?

Reply:
The existing `<>` is just duplicating the signature in both directions specified by the header
`alert tcp 1.2.3.4 any <> 5.6.7.8 80 (sid: 1; etc...)`
becomes
```
alert tcp 1.2.3.4 any -> 5.6.7.8 80 (sid: 1; etc...)
alert tcp 5.6.7.8 80 -> 1.2.3.4 any (sid: 1; etc...)
```
I do not know the use case for it...

This PR with `=>` allows to match on both sides of a transaction like
`alert http any any => any any (msg:"matching both uri and status"; sid: 1; http.uri; content: "/download"; http.stat_code; content: "200";)`
matching the URI from the client, and the status from the server

Let me know if you have a better wording...

Reply:
Thank you very much!
After reading up what bidirectional means, I think it makes more sense for the `<>` operator as we use the same rule in both the directions.
Suggestions for your work:
1. transaction scoped rule (flags and varnames could be like `*tx_scope*`)
2. full transaction match rule (flags and varnames could be like `*full_tx_match*`)
3. transaction span rule (flags and varnames could be like `*tx_span*`)

Interested in hearing your and others' thoughts.
Also, we should make sure that if we make changes, the `<>` varnames and flags are all named alike as should be the new ones, I think.

Reply:
Thanks Shivani for the proposals.
I think it is still good to use somehow the word direction to indicate that these rules do not have a direction like others...